charts/backup/resources/backup.sh

```diff
@@ -96,11 +99,144 @@ then
   s3cmd put $FILE_NAME.gpg $BUCKET_NAME
   echo "Finished Postgres backup"
 fi
-echo "Finished backup process"
+
+if [ -n "$BUDIBASE_ADMIN_PASS" ]
```

Comment:
wow sometimes a simple mysqldump is quite nice lol

I found budi backups --export - this is not an option here? Then Budibase would handle the backup. 
or npm install -g @cloudant/couchbackup (from IBM/Cloudant).

that looks much more easy.